source/ProSymbolEditor/Models/Symbol Attributes/DisplayAttributes.cs

```diff
@@ -0,0 +1,380 @@
+﻿/*******************************************************************************
```

Comment:
Probably should have avoided spaces in the folder name - just to avoid headaches later